lib/manager/sbt/update.ts

```diff
@@ -0,0 +1,56 @@
+import { ReleaseType, inc } from 'semver';
+import { logger } from '../../logger';
+
+export function bumpPackageVersion(
+  content: string,
+  currentValue: string,
+  bumpVersion: ReleaseType | string
+): string {
+  logger.debug(
+    {
+      bumpVersion,
+      currentValue,
+    },
+    'Checking if we should bump build.sbt version'
+  );
+  let newVersion: string;
+  try {
+    logger.info({
+      content,
+      currentValue,
+      bumpVersion,
+    });
+
+    newVersion = inc(currentValue, bumpVersion as ReleaseType);
+
+    if (!newVersion) {
+      throw new Error('Version incremental failed');
```

Comment:
log a warning and return, no error necessary

Reply:
this means that the `try-catch` statement is not needed anymore